tests/unittest.cpp

```diff
@@ -102,12 +102,12 @@ TEST_CASE("Helixpath") {  /// Helix path test
   SurfaceCylinder cyl2(papas::Position::kEcalOut, 2., 1.);
   double field = 3.8;
   PFParticle particle(211, -1, TLorentzVector{2., 0, 1, 5}, 1, 'r', TVector3{0, 0, 0}, field);
-  HelixPropagator helixprop(3.8);
+  HelixPropagator helixprop;
   //(particle.p4(), {0,0,0}, 3.8, -1);
-  helixprop.propagateOne(particle, cyl1);
+  helixprop.propagateOne(particle, cyl1, field);
```

Comment:
That is a nice idea. I have done so and removed all the other field arguments noted in your comments.